### PR TITLE
Fix building with latest Bevy

### PR DESCRIPTION
**Pull request by billyb2**
I just made some slight modifications to the library so that it would build on the latest Bevy git commit (and so the code will be more prepared for v0.6).

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -22,7 +22,7 @@ default = [
 
 [dependencies]
 winit = {version = "0.25.0", features = ["web-sys"]}
-bevy = { version="0.5.0", default-features=false }
+bevy = { git = "https://github.com/bevyengine/bevy", default-features=false }
```
Comment (TomPridham) [inline Cargo.toml:25]: i think you will need to specify the branch name(at least, i have to when trying to use the git repo directly). cargo assumes `master` will be the default branch and bevy's is `main`

```suggestion
bevy = { git = "https://github.com/bevyengine/bevy", branch = "main", default-features=false }
```

Comment (billyb2) [reply Cargo.toml]: Thank you, I wasn't aware of that.

File: src/lib.rs [hunk 2]
```diff
@@ -26,6 +26,7 @@ use bevy::render::{
     shader::{Shader, ShaderStage},
     RenderStage,
 };
+use bevy::prelude::Plugin;
```
Comment (mrk-its) [inline src/lib.rs:29]: it seems it is unused now

File: Cargo.toml [hunk 3]
```diff
@@ -22,7 +22,7 @@ default = [
 
 [dependencies]
 winit = {version = "0.25.0", features = ["web-sys"]}
-bevy = { version="0.5.0", default-features=false }
+bevy = { git = "https://github.com/bevyengine/bevy", branch = "main", default-features=false }
```
Comment (mrk-its) [inline Cargo.toml:25]: as you are changing bevy version here please remove whole `[patch.crates-io]` section below

**Review by mrk-its — CHANGES_REQUESTED** [review]
Please rebase on current master - I've just renamed `App::build` to `App::new` in examples -  it seems all examples compile after that :+1:

**Comment by billyb2** [comment]
Would you mind if I upgraded web-sys as well

**Comment by mrk-its** [comment]
> Would you mind if I upgraded web-sys as well
Let's upgrade

**Comment by billyb2** [comment]
Did it. However, I can't upgrade web-sys, as it breaks builds (dependency conflict).

**Comment by mrk-its** [comment]
> Did it. However, I can't upgrade web-sys, as it breaks builds (dependency conflict).

thanks, taking a look

**Comment by mrk-its** [comment]
Merged, thanks!

**Comment by billyb2** [comment]
Thank you!

**Pull request merged**